charts/icm-as/templates/ssl-certificate-spc.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.sslCertificateRetrieval.enabled }}
-apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+apiVersion: secrets-store.csi.x-k8s.io/v1
```

Comment:
Question for the reviewers: The alpha version has been deprecated. Is this a breaking change? Should I leave it out? I also couldn't really test the new version in production, since my the certificate retrieval process works differently in my cluster.

Reply:
Maybe this wouldn't belong to this pull request. Please create a separate one. Also I assume this would be a breaking change as it needs an update of the CSI provider.

Reply:
See #385